frontend-nextjs/features/(dashboard)/FileUpload.tsx

```
@@ -1,122 +1,319 @@
+					throw new Error("アップロードに失敗しました");
+					success = false;
 				}
-			} else {
-				setMessage("ファイルのアップロードに失敗しました。");
+			} catch (error) {
+				alert(
+					`エラー: ${error instanceof Error ? error.message : "不明なエラーが発生しました"}`,
+				);
+				success = false;
+			} finally {
+				setIsUploading(false);
```

Biome

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)